### DISCUSSION TITLE
Change in official school email domain

### DISCUSSION THREAD

**Discussion by JessieRoeder** [community/community#41650] [GitHub Education]
I tried to submit this issue through GitHub Support, but I got frustratingly auto-directed to post it here. It is not a basic platform issue, or anything that anyone outside of staff is likely to be able to help with, but here we are anyway.

To renew my teacher status, the application requires me to use an @johnadamsnyc.org email to prove that I belong to my school. These emails have been discontinued. All NYC public schools are eventually switching over to @schools.nyc.gov emails for their faculty and @nycstudents.net emails for students, and my school switched over this year. I need someone to update the domain for my school to be @schools.nyc.gov emails for faculty, and @nycstudents.net emails for students.

**Comment by samus-aran** [comment]
Hi @JessieRoeder 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐ 

I've reached out to the team and they have updated the domain for your particular school. If you can try applying again and let us know how you get on 👍

**Reply by FormicoRofo** [reply]
> Hi @JessieRoeder 👋
> 
> We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐
> 
> I've reached out to the team and they have updated the domain for your particular school. If you can try applying again and let us know how you get on 👍

@samus-aran 
Hi ! I have the same problem (student at ETML - Ecole des Métiers de Lausanne), it asks for @etml.educanet2.ch but the educanet2 system was taken down two years ago for the new @eduvaud.ch now used for all cantonal schools in the canton. (for example the Haute Ecole d'Ingénierie et de Gestion du Canton de Vaud – HEIG-VD has it's domain updated).

Would it be possible to do something ? thanks !

**Reply by mat345st** [reply]
> Hi @JessieRoeder 👋
> 
> We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐
> 
> I've reached out to the team and they have updated the domain for your particular school. If you can try applying again and let us know how you get on 👍

@samus-aran 
Hi, I have the same issue. My request for the Student Developer Pack as student of Kopernikus-Gymnasium Neubeckum got rejected with "Attendees of this school must verify and use their academic email address.". It requires me to use a @kopernikus-neubeckum.de mail, but students of my school have @beckum.schule mails. Would it be possible to update this too?

**Reply by lilmistake** [reply]
Can I get some help on this as well please. My school does not give students any academic emails but my app got rejected with "Attendees of this school must verify and use their academic email address." and I have tried every possible way to prove that I am a student.

**Reply by emosy** [reply]
I have a similar issue. I was told to apply using a specific college for my school since the colleges offer subdomains for their employees. However, all students use @gatech.edu emails. I applied and manually entered Georgia Tech but was rejected and told to use my college-specific email. I do not have an @cc.gatech.edu email. Please fix this or let me contact someone more easily!

**Reply by ms-studio** [reply]
I have the same issue: after a domain change of my school, I need to get the new email domain being recognized by Github. My ticket ID : 1984983. Thanks for looking into it 🙏
